App.js

```diff
@@ -1,77 +1,48 @@
-import React, {useState, useEffect} from 'react';
-import {StyleSheet, View, ScrollView, RefreshControl} from 'react-native';
-import {NavBar} from './src/components/Nav-bar';
-import {NavBarDetails} from './src/components/Nav-bar-details';
-import {SearchInput} from './src/components/Search-input';
-import {Card} from './src/components/Card';
-import mockCards from './src/components/mockCards';
-import {CardLarge} from './src/components/Card-Large';
-
-const BASE_URL = 'https://demo.spreecommerce.org';
-const API_PATH = `${BASE_URL}/api/v2/storefront/products`;
-
-const wait = timeout => {
-  return new Promise(resolve => setTimeout(resolve, timeout));
-};
+import React from 'react';
+import {NavigationContainer} from '@react-navigation/native';
+import {createNativeStackNavigator} from '@react-navigation/native-stack';
+import {MainScreen} from './src/pages/Main-screen';
+import {Product} from './src/pages/Product';
```

Comment:
This is not working 😃  Please check file name. It should start with lowercase letter